packages/gatsby/src/schema/infer/add-inferred-fields.js

```diff
@@ -36,11 +36,21 @@ const addInferredFields = ({
 
   if (deprecatedNodeKeys.size > 0) {
     reportOnce(
-      `The ___NODE convention is deprecated. Please use the @link directive instead.\nType: ${typeComposer.getTypeName()}, Keys: ${Array.from(
-        deprecatedNodeKeys
-      ).join(`, `)}\nMigration: https://gatsby.dev/node-convention-deprecation`,
+      `The ___NODE convention is deprecated. Please use the @link directive instead.\nMigration: https://gatsby.dev/node-convention-deprecation`,
       `verbose`
     )
+    reportOnce(
+      `Set process.env.GATSBY_LOG_NODE_DEPRECATED_NODE_FIELDS to "true" to see which fields are using the deprecated ___NODE convention.`,
+      `verbose`
+    )
+
+    if (process.env.GATSBY_LOG_NODE_DEPRECATED_NODE_FIELDS === `true`)
```

Comment:
If I remember correctly there's a util that should be used for checking env vars instead of checking directly. Is that right? I couldn't find it